src/explore-education-statistics-admin/src/pages/publication/components/__tests__/PublicationPublishedReleases.test.tsx

```diff
@@ -318,7 +318,8 @@ describe('PublicationPublishedReleases', () => {
           {
             ...testRelease1,
             permissions: {
-              ...testRelease1.permissions,
+              // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+              ...testRelease1.permissions!,
```

Comment:
If we separate the permissions out into a separate variable with an explicit type, we don't need this non-null assertion